src/jlgen.jl

```diff
@@ -749,3 +749,14 @@ function compile_method_instance(@nospecialize(job::CompilerJob))
 
     return llvm_mod, compiled
 end
+
+function CC.typeinf(interp::GPUInterpreter, frame::CC.InferenceState)
```

Comment:
Shouldn't do things without drinking coffee first. This should needs a version bound to limit it for 1.11.

In 1.11-beta2 it should not be necessary anymore